CRITICAL - 2023-06-14 00:17:13 --> "alfa_numeric_space" is not a valid rule.
in SYSTEMPATH\Validation\Validation.php on line 329.
 1 SYSTEMPATH\Validation\Validation.php(329): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('alfa_numeric_space')
 2 SYSTEMPATH\Validation\Validation.php(179): CodeIgniter\Validation\Validation->processRules('address', 'address', null, [...], [...])
 3 APPPATH\Filters\Filter.php(41): CodeIgniter\Validation\Validation->run()
 4 APPPATH\Controllers\Auth.php(65): App\Filters\Filter::validateDatesUserRegister(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->registerUser()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-06-14 01:55:43 --> PaswordHash --> : $2y$10$px6S.f103PDV7w18PZKUEefxSB3nQNPGEH0SPrcaPVxAJ5C0up.Pu
INFO - 2023-06-14 02:05:04 --> --Soy el filtro encontre estos permisos : 
INFO - 2023-06-14 02:25:10 --> PaswordHash --> : $2y$10$Mn7daYprQDEjywJdsE3BDeI0L.KJrCSKtGJOVNC87FZ32ElNP.q2m
ERROR - 2023-06-14 02:25:10 --> ErrorException: pg_query(): Query failed: ERROR:  la sintaxis de entrada no es válida para tipo date: «»
LINE 1: ...'usu@example.com', 'Axel K 67112', '', '', '', NULL, '', '')
                                                                    ^ in D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Database\Postgre\Connection.php:152
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\XAMPP\\htdocs...', 152)
#1 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Database\Postgre\Connection.php(152): pg_query(Object(PgSql\Connection), 'INSERT INTO "us...')
#2 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(692): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "us...')
#3 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "us...')
#4 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO "us...', Array, false)
#5 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#6 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#8 D:\XAMPP\htdocs\codeigniter4\app\Controllers\Auth.php(72): CodeIgniter\Model->insert(Array, false)
#9 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Auth->registerUser()
#10 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#11 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\XAMPP\htdocs\codeigniter4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-06-14 02:25:10 --> pg_query(): Query failed: ERROR:  la sintaxis de entrada no es válida para tipo date: «»
LINE 1: ...'usu@example.com', 'Axel K 67112', '', '', '', NULL, '', '')
                                                                    ^
in SYSTEMPATH\Database\Postgre\Connection.php on line 152.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Query failed: ERROR:  la sintaxis de entrada no es válida para tipo date: «»
LINE 1: ...\'usu@example.com\', \'Axel K 67112\', \'\', \'\', \'\', NULL, \'\', \'\')
                                                                    ^', 'D:\\XAMPP\\htdocs\\codeigniter4\\vendor\\codeigniter4\\framework\\system\\Database\\Postgre\\Connection.php', 152)
 2 SYSTEMPATH\Database\Postgre\Connection.php(152): pg_query(Object(PgSql\Connection), 'INSERT INTO "users" ("password", "email", "username", "name", "lastname", "address", "gender", "phonenumber", "birthdate") VALUES (\'$2y$10$Mn7daYprQDEjywJdsE3BDeI0L.KJrCSKtGJOVNC87FZ32ElNP.q2m\', \'usu@example.com\', \'Axel K 67112\', \'\', \'\', \'\', NULL, \'\', \'\')')
 3 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "users" ("password", "email", "username", "name", "lastname", "address", "gender", "phonenumber", "birthdate") VALUES (\'$2y$10$Mn7daYprQDEjywJdsE3BDeI0L.KJrCSKtGJOVNC87FZ32ElNP.q2m\', \'usu@example.com\', \'Axel K 67112\', \'\', \'\', \'\', NULL, \'\', \'\')')
 4 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "users" ("password", "email", "username", "name", "lastname", "address", "gender", "phonenumber", "birthdate") VALUES (\'$2y$10$Mn7daYprQDEjywJdsE3BDeI0L.KJrCSKtGJOVNC87FZ32ElNP.q2m\', \'usu@example.com\', \'Axel K 67112\', \'\', \'\', \'\', NULL, \'\', \'\')')
 5 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO "users" ("password", "email", "username", "name", "lastname", "address", "gender", "phonenumber", "birthdate") VALUES (:password:, :email:, :username:, :name:, :lastname:, :address:, :gender:, :phonenumber:, :birthdate:)', [...], false)
 6 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 7 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 8 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 9 APPPATH\Controllers\Auth.php(72): CodeIgniter\Model->insert([...], false)
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->registerUser()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-06-14 02:38:53 --> PaswordHash --> : $2y$10$TMie1XI9zmkkWOkHZEKl8uxTrRnouz8W.HgmBGEb.Zx8pOnj54wL.
INFO - 2023-06-14 02:59:42 --> PaswordHash --> : $2y$10$DHLz.K0ZF/2XJMMXu/QeZOBv7kNOZksXkru271zlGd8Xhun4c/I6O
WARNING - 2023-06-14 02:59:42 --> [DEPRECATED] preg_match(): Passing null to parameter #2 ($subject) of type string is deprecated in SYSTEMPATH\Email\Email.php on line 475.
 1 SYSTEMPATH\Email\Email.php(475): preg_match('/\\<(.*)\\>/', null, null)
 2 APPPATH\Controllers\Auth.php(87): CodeIgniter\Email\Email->setFrom(null, 'Es un correo de Tu aplicacion')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Auth->registerUser()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-06-14 03:02:10 --> PaswordHash --> : $2y$10$qQ0Qh/DmDoACHB4VltYcKO1Q4996vsYTBDs8yqtEkTUxp.prAzUaW
INFO - 2023-06-14 03:26:05 --> PaswordHash --> : $2y$10$U2IzkjZEAJksOz6KPuzW1OGsiIPqELb.x9LK2XpCzBzwLwNgxDNya
ERROR - 2023-06-14 03:26:07 --> Email: sendWithMail throwed ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() in D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Email\Email.php:1735
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed ...', 'D:\\XAMPP\\htdocs...', 1735)
#1 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Email\Email.php(1735): mail('axelx33@hotmail...', '=?UTF-8?Q?Prueb...', 'Este es un corr...', 'Date: Wed, 14 J...', '-f remitente@ex...')
#2 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Email\Email.php(1667): CodeIgniter\Email\Email->sendWithMail()
#3 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Email\Email.php(1565): CodeIgniter\Email\Email->spoolEmail()
#4 D:\XAMPP\htdocs\codeigniter4\app\Controllers\Auth.php(93): CodeIgniter\Email\Email->send()
#5 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Auth->registerUser()
#6 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\XAMPP\htdocs\codeigniter4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2023-06-14 03:26:07 --> Email: Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.
ERROR - 2023-06-14 03:26:07 --> 
INFO - 2023-06-14 03:26:55 --> PaswordHash --> : $2y$10$Rt182kT8rpuKR67GgC1Ai.fiemSiCoqYFn35hxUjFoLX5THoNIt46
ERROR - 2023-06-14 03:26:57 --> Email: sendWithMail throwed ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() in D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Email\Email.php:1735
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed ...', 'D:\\XAMPP\\htdocs...', 1735)
#1 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Email\Email.php(1735): mail('axelx33@hotmail...', '=?UTF-8?Q?Prueb...', 'Este es un corr...', 'Date: Wed, 14 J...', '-f axelx33@hotm...')
#2 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Email\Email.php(1667): CodeIgniter\Email\Email->sendWithMail()
#3 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\Email\Email.php(1565): CodeIgniter\Email\Email->spoolEmail()
#4 D:\XAMPP\htdocs\codeigniter4\app\Controllers\Auth.php(93): CodeIgniter\Email\Email->send()
#5 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Auth->registerUser()
#6 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 D:\XAMPP\htdocs\codeigniter4\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\XAMPP\htdocs\codeigniter4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2023-06-14 03:26:57 --> Email: Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.
ERROR - 2023-06-14 03:26:57 --> 
INFO - 2023-06-14 18:27:15 --> --Errores de Validacion :  The email field must contain a unique value.
